### PR TITLE
Bug 1096276 - Docs: docker: document how to re-enable IP packet forwarding

### PR DESCRIPTION
**Pull request by lproven**
### Description
Bugfix/bsc#1096276 addresses long-standing bug requesting that our docs link to the upstream documentation on the interactions between Docker containers and the system firewall. Also did a copy-edit on the Docker chapter while I was in there.

### Checklist
* Check all items that apply.

*Are backports required?*

- [x] To maintenance/SLE15SP1
- [x] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/docker_dockerizing_application.xml [hunk 1]
```diff
@@ -10,35 +10,34 @@
...
-  build applications. There are several types of applications that are
-  suitable to run inside a &docker; container like daemons, Web pages or
-  applications that expose ports for communication. You can use &deng; to
-  automate building and deployment processes by adding the build process
-  into a &docker; image, then building the image and then running containers
-  based on that image.
+  &deng; is a technology that can help minimize the resources used to run
+  applications. Several types of applications are suitable to be run inside
+  &docker; containers, such as daemons, Web pages or applications that expose
+  IP ports for communications. You can use &deng; to automate the building and
```
Comment (tbazant) [inline xml/docker_dockerizing_application.xml]: quite a long sentence....i lost attention at its end :-)

File: xml/docker_installation.xml [hunk 2]
```diff
@@ -11,36 +11,36 @@
...
    <para>
-    Starting with &deng; 1.12, the container orchestration is now an integral
+    Starting with &deng; 1.12, container orchestration is now an integral
     part of &deng;. Even though this feature is available in
     &productname;, it is not supported by &suse; and is only provided as a
-    technical preview. Use Kubernetes for &docker; container orchestration, for
-    details refer to the <link
-    xlink:href="http://kubernetes.io/docs/getting-started-guides/kubeadm/">Kubernetes
-    documentation</link>.
+    technical preview. Use Kubernetes for &docker; container orchestration; for
```
Comment (tbazant) [inline xml/docker_installation.xml]:
```suggestion
    technical preview. Use Kubernetes for &docker; container orchestration. For
```

File: xml/docker_installation.xml [hunk 3]
```diff
@@ -11,36 +11,36 @@
...
+  
   <procedure>
    <title>Enabling the Container Module Using &yast;</title>
    <step>
-
+    
     <para>
      Start &yast;, and select <menuchoice> <guimenu>Software</guimenu>
-     <guimenu> Software Repositories</guimenu> </menuchoice>.
+      <guimenu> Software Repositories</guimenu> </menuchoice>.
```
Comment (tbazant) [inline xml/docker_installation.xml]:
```suggestion
      <guimenu>Software Repositories</guimenu> </menuchoice>.
```

File: xml/docker_installation.xml [hunk 4]
```diff
@@ -220,24 +221,40 @@
...
    </para>
   </sect2>
-  </sect1>
-
-  <sect1 xml:id="sec-docker-setup-updates">
+  
+  <sect2 xml:id="sec-docker-setup-net-docs">
+   <title>How the &deng; Interacts with <command>iptables</command></title>
+   <para>
+    To learn more about &docker; containers interact with each other and the
```
Comment (tbazant) [inline xml/docker_installation.xml]:
```suggestion
    To learn more about how &docker; containers interact with each other and the
```

File: xml/docker_registry.xml [hunk 5]
```diff
@@ -56,41 +55,49 @@
...
-     The authentication component: This is used to protect the images stored
-     inside of &dhub;. It validates all push, pull and search requests.
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>The authentication component</term>
+    <listitem>
+     <para>
+     This is used to protect the images stored inside of &dhub;. It validates
+     all push, pull and search requests.
```
Comment (tbazant) [inline xml/docker_registry.xml]:
```suggestion
     all push, pull, and search requests.
```

File: xml/docker_dockerizing_application.xml (hunk 1 above)
Comment (lproven) [reply xml/docker_dockerizing_application.xml]: OK, it is a bit... but slightly shorter than it was before...?

Comment (tbazant) [reply xml/docker_dockerizing_application.xml]: yes, it's better now

**Pull request merged**